docs/src/main/sphinx/release/release-368.md

```diff
@@ -0,0 +1,51 @@
+# Release 368 (xx Jan 2022)
+
+## General
+
+* Improve performance of arithmetic operations involving decimals with precision
```

Comment:
We usually sort items in this order:

* New features / improvements
* Performance improvements
* Bug fixes

Reply:
Done